### PR TITLE
Space being wasted as margin. Made margin 0 so that table can be as large as possible

### PR DESCRIPTION
**Pull request by namboodiri**

### CODE REVIEW

**Comment by abhaga** [comment]
Can you also set `overflow-x: visible` on the `table-responsive` class as well? `table-responsive` class sets it to `auto` by default which is not working due to very long strings in the data. I have tested it on Desktop but not able to do so on mobile right now.

**Comment by namboodiri** [comment]
ok am on it.

**Comment by namboodiri** [comment]
But i think, if the width of the table is 100%, we don't need the overflow to be visible, it should be rather be scroll right?

**Comment by abhaga** [comment]
I tried scroll, doesn't seem to work.

**Comment by biswaz** [comment]
Can you guys ping me on slack

**Comment by vrthra** [comment]
+1 -- tested.

**Comment by bobinson** [comment]
@biswaz - please open up slack for people who are willing to pick up the issues.

+1 tested and works fine.

**Comment by jishnunand** [comment]
@biswaz Tested the changes in mobile works fine. If possible create a slack channel.

**Comment by namboodiri** [comment]
Are we merging this?

**Comment by sibStephen** [comment]
There are still some responsive issues on smaller devices (between 320px to 480px) but if working on all other devices then it's good to go..

**Comment by bhumilharia** [comment]
Testing

**Comment by bhumilharia** [comment]
Tested on Chrome and Firefox, desktop and mobile. Tables show properly on mobile.

When clicking on "more details" on `requests` page, the details page introduces a horizontal scroll (I assume this is intended?).

If intended, this is tested.

**Comment by vrthra** [comment]
Marking this as tested, but would like to get a response on @bhumilharia's question by the dev.

**Comment by ajaykanayil** [comment]
Guys, keep large sentences/ paragraphs under 50-70 charactors width so that it is readable... Anything more than thaty would be hard to follow...

**Comment by NanduKMG** [comment]
Why don't we use "container-fluid" instead of using "container" with width:100% ?
https://getbootstrap.com/docs/3.3/css/#overview-container
Please check into this.Cheers

**Comment by namboodiri** [comment]
@bhumilharia, that is intended and not part of this PR
